opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -35,22 +35,35 @@ def setUpClass(cls):
         cls.serialized_span_id = b3_format.format_span_id(
             trace.generate_span_id()
         )
+        cls.serialized_parent_span_id = b3_format.format_span_id(
+            trace.generate_span_id()
+        )
 
     def test_extract_multi_header(self):
         """Test the extraction of B3 headers."""
         carrier = {
             FORMAT.TRACE_ID_KEY: self.serialized_trace_id,
             FORMAT.SPAN_ID_KEY: self.serialized_span_id,
+            FORMAT.PARENT_SPAN_ID_KEY: self.serialized_parent_span_id,
             FORMAT.SAMPLED_KEY: "1",
         }
-        span_context = FORMAT.extract(get_as_list, carrier)
+        parent_span = trace.Span(
+            "parent", FORMAT.extract(get_as_list, carrier)
+        )
+        child_span = trace.Tracer().start_span("child", parent=parent_span)
```

Comment:
I believe you could just instantiate a raw Span object yourself from the SDK, rather than use a whole trace to do so.

Reply:
Yes, fixed!